projects/GEDKeeper2/GKUI/MediaViewerWin.cs

```diff
@@ -184,5 +184,18 @@ private void MediaViewerWin_KeyDown(object sender, KeyEventArgs e)
                 Close();
             }
         }
+        void MediaViewerWinFormClosing(object sender, FormClosingEventArgs e)
+        {
+            MultimediaKind mmKind = GKUtils.GetMultimediaKind(fFileRef.MultimediaFormat);
+            switch (mmKind)
+            {
+                case MultimediaKind.mkVideo:
+                    {
+                        ((MediaPlayer)fViewer).btnStop_Click(null, null);
+                        break;
+                    }
+            }
+        }

```

Comment:
You plan to handle some additional "media kinds", don't you? This is the reason you have used `swtich` statement, and not `if` one.